tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -135,17 +135,31 @@ export async function assertIdentityRemoved(
         const eventData = events[index].data;
         const who = eventData.account.toHex();
 
-        // Check idGraph
         assert.equal(
-            eventData.idGraph,
```

Comment:
I'm not sure why we had this check as this event doesn't contain `idGraph` information.